src/main/java/com/linkedin/kmf/services/configs/KafkaMetricsReporterServiceConfig.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright 2016 LinkedIn Corp. Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.linkedin.kmf.services.configs;
+
+import org.apache.kafka.common.config.AbstractConfig;
+import org.apache.kafka.common.config.ConfigDef;
+
+import java.util.Arrays;
+import java.util.Map;
+
+import static org.apache.kafka.common.config.ConfigDef.Range.atLeast;
+
+public class KafkaMetricsReporterServiceConfig extends AbstractConfig {
+
+  private static final ConfigDef CONFIG;
+
+  public static final String REPORT_METRICS_CONFIG = CommonServiceConfig.REPORT_METRICS_CONFIG;
+  public static final String REPORT_METRICS_DOC = CommonServiceConfig.REPORT_METRICS_DOC;
+
+  public static final String REPORT_INTERVAL_SEC_CONFIG = CommonServiceConfig.REPORT_INTERVAL_SEC_CONFIG;
+  public static final String REPORT_INTERVAL_SEC_DOC = CommonServiceConfig.REPORT_INTERVAL_SEC_DOC;
+
+  public static final String ZOOKEEPER_CONNECT_CONFIG = CommonServiceConfig.ZOOKEEPER_CONNECT_CONFIG;
+  public static final String ZOOKEEPER_CONNECT_DOC = CommonServiceConfig.ZOOKEEPER_CONNECT_DOC;
+
+  public static final String BOOTSTRAP_SERVERS_CONFIG = CommonServiceConfig.BOOTSTRAP_SERVERS_CONFIG;
+  public static final String BOOTSTRAP_SERVERS_DOC = CommonServiceConfig.BOOTSTRAP_SERVERS_DOC;
+
+  public static final String TOPIC_CONFIG = CommonServiceConfig.TOPIC_CONFIG;
+  public static final String TOPIC_DOC = CommonServiceConfig.TOPIC_DOC;
+
+  public static final String TOPIC_REPLICATION_FACTOR = "reporter.kafka.topic.replication.factor";
```

Comment:
Just one last minor comment. Since this service uses the config `report.interval.sec`, can we name this config as `reporter.kafka.topic.replication.factor` for consistency?

Reply:
Okay, I'll update to `report.kafka.topic.replication.factor`